arangod/Transaction/Methods.cpp

```diff
@@ -540,15 +540,12 @@ transaction::Methods::~Methods() {
         // regular life cycle. we want now to properly clean up and count them.
         _state->updateStatus(transaction::Status::COMMITTED);
       } else {
-        try {
-          this->abort();
-          TRI_ASSERT(_state->status() != transaction::Status::RUNNING);
```

Comment:
Do we want to keep this assertion?

Reply:
We have the same assertion a few lines below. I thought that suffices - WDYT?

Reply:
Ah, I missed that in the diff. All good!